pkg/test/framework/components/apps/kube.go

```diff
@@ -632,11 +631,6 @@ func (a *kubeApp) Call(e AppEndpoint, opts AppCallOptions) ([]*echo.ParsedRespon
 		return nil, fmt.Errorf("supplied endpoint was not created by this environment")
 	}
 
```

Comment:
I am not sure why this PR removes the following code segment:
        // Normalize the count.	
	if opts.Count <= 0 {	
		opts.Count = 1	
	}

Reply:
It's duplicated. There is a exact same code 1-2 lines below this code.